js/components/calendar.vue

```diff
@@ -13,39 +13,39 @@
 </style>
 
 <template>
-<div class="calendar datepicker" v-class="view">
+<div class="calendar datepicker" :class="[ 'view' ]">
     <div class="datepicker-{{view}}">
         <table class="table table-condensed">
             <thead>
                 <tr>
                     <th class="prev" style="visibility: visible;"
-                        v-on="click: previous">«</th>
+                        @click="previous">«</th>
                     <th colspan="5" class="datepicker-switch"
-                        v-on="click: zoomOut">{{rangeDisplay}}</th>
+                        @click="zoomOut">{{rangeDisplay}}</th>
                     <th class="next" style="visibility: visible;"
-                        v-on="click: next">»</th>
+                        @click="next">»</th>
                 </tr>
                 <tr v-if="view == 'days'">
-                    <th class="dow" v-repeat="days">{{$value}}</th>
+                    <th class="dow" v-for="day in days">{{day}}</th>
                 </tr>
             </thead>
             <tbody v-show="view == 'days'">
-                <tr v-repeat="week:currentMonthDays">
+                <tr v-for="week in currentMonthDays">
                     <td class="day"
-                        v-repeat="day:week"
-                        v-on="click: pickDay(day)"
-                        v-class="
-                            old: isOld(day),
-                            new: isNew(day),
-                            today: day.isSame(today, 'day'),
-                            active: day.isSame(selected, 'day')
-                        ">{{ day.date() }}</td>
+                        v-for="day in week"
+                        @click="pickDay(day)"
+                        :class="{
+                            'old': isOld(day),
+                            'new': isNew(day),
+                            'today': day.isSame(today, 'day'),
+                            'active': day.isSame(selected, 'day')
+                        }">{{ day.date() }}</td>
                 </tr>
             </tbody>
             <tbody v-show="view == 'months'">
                 <tr>
                     <td colspan="7">
-                        <span class="month" v-repeat="months" v-on="click: pickMonth($index)">{{$value}}
+                        <span class="month" v-for="month in months" @click="pickMonth($index)">{{month}}
```

Comment:
I think that we should use the `(index, val) in arr` syntax to avoid implicit/magic stuff.